src/styles/pages/_Home.module.scss

```diff
@@ -1,24 +1,35 @@
 .home {
   > section {
     margin-bottom: 8rem;
-
-    &:not(.posts) {
-      padding: 0 3rem;
-    }
   }
 }
 
 .heading {
+  font-size: 4.8rem;
+  text-align: center;
   margin-bottom: 1rem;
 }
 
 .description {
-  max-width: 50%;
+  max-width: 55%;
```

Comment:
We should have media queries for this property so reading on mobile is not so restricted. Maybe something like:

```scss
.description {
   // ...
  max-width: 100%;

  @media (min-width: $breakpoint-small) {
    max-width: 55%;
  }
}
```